### PR TITLE
Pull Request for List structure : To-Do-List

### PR DESCRIPTION
**Pull request by anita00001**
In this branch, we have created a basic list structure for a To-Do-List application.

Features:
✔No Linters
✔Descriptive documentation
✔Best use of HTML, CSS, and Javascript
✔Use of Webpack

Key features:
⏳Add a task in to-do-list
🧭Mark the completed task as checked

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+      ...this.taskList,
+      { description },
+    ];
+    this.updateStorage(updateTask);
+  }
+
+  // removeTask(id) {
+  //   // const updateTask = this.taskList.filter((it) => it.id !== id);
+  //   this.updateStorage(updateTask);
+  // }
```
Comment (AnselemOdims) [inline src/index.js]: - [x] Good job working on this, but please kindly remove the code comments as it pollutes the codebase.

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+const tasks = new ToDo();
+let taskListArray = tasks.getFromStorage();
+
+// let counter = 0;
+// function next() {
+//   counter += 1;
+//   return counter;
+// }
```
Comment (AnselemOdims) [inline src/index.js]: - [x] As mentioned, please kindly remove the code comments as it pollutes the codebase.

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+const showTask = () => {
+  showToDo.innerHTML = '';
+  // let currentNumber = 1;
+  // currentNumber = next(currentNumber);
+  // currentNumber -= 1;
```
Comment (AnselemOdims) [inline src/index.js]: - [x] As mentioned, please kindly remove the code comments as it pollutes the codebase.

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @anita00001, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [ ] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+// Local Storage
+class ToDo {
+  constructor() {
+    this.taskList = JSON.parse(localStorage.getItem('storage-task')) || [];
+  }
+
+  addTask(description) {
+    const updateTask = [
+      ...this.taskList,
+      { description },
```
Comment (binhussen) [inline src/index.js:29]: - [Optional] Great job for implementing previous code reviewer suggestions, however, you have missed one requirement, which is your object has to include an index and completed :+1:  please update to track the completion and identity each to do the task as the below sample
```
const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
```

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+  align-items: center;
+  justify-content: center;
+}
+
+main {
+  width: 90%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 2px solid black;
```
Comment (binhussen) [inline src/style.css:19]: - [Optional] you added a border and your todo list cards are larger than the given design please update it to have a better design :+1:

**Review by binhussen — APPROVED** [review]
Hi @anita00001 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- All linter checks passed ✔️
- Descriptive pull request ✔️
- Used GitHub flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**